tutorials/R/download-explore/NEON-download-explore.Rmd

```diff
@@ -130,67 +127,125 @@ called **stackedFiles**, which should contain five files:
 
 We'll look at these files in more detail below.
 
-## Download data using neonUtilities: zipsByProduct()
+## Download files and load directly to R: loadByProduct()
 
-The function `zipsByProduct()` is a wrapper for the NEON API, it 
-downloads zip files for the data product specified and stores them in 
-a format that can then be passed on to `stackByTable()`.
+In the section above, we downloaded a .zip file from the data portal to
+our downloads folder, then used the stackByTable() function to transform
+those data into a usable format. However, there is a faster way to load
+data directly into the R Global Environment using `loadByProduct()`.
 
-One of the inputs to `zipsByProduct()` is the data product ID, or 
-DPID, of the data you want to download. The DPID can be found on the 
-<a href="http://data.neonscience.org/data-products/explore" target="_blank">Explore Data Products page</a>.
-It will be in the form DP#.#####.###; e.g., the DPID of PAR, downloaded  
-above, is DP1.00024.001.
+The most popular function in `neonUtilities` is `loadByProduct()`. 
+This function downloads data from the NEON API, merges the site-by-month 
+files, and loads the resulting data tables into the R environment, 
+assigning each data type to the appropriate R class. It combines the 
+actions of the `zipsByProduct()`, `stackByTable()`, and `readTableNEON()` 
```

Comment:
Need to modify this section a little, since this tutorial doesn't describe zipsByProduct() anymore

Reply:
I ended up removing much of this text and combining with the following paragraph, then pointing to the neonUtilities tutorial below (see third comment below)